lib/ProductOpener/Display.pm

```diff
@@ -1570,12 +1570,19 @@ sub display_list_of_tags($$) {
 					if ($region eq 'UK') {
 						$region = 'GB';
 					}
-					$countries_map_links .=  '"' . $region . '": "' . $product_link . "\",\n";
-					$countries_map_data .= '"' . $region . '": ' . $products . ",\n";
+
+					$countries_map_links->{$region} = $product_link;
```

Comment:
Shouldn't we keep only the first $product_link (which should be the most common one as the list is in descending order). In that case maybe we should only keep the first count and not add the count for other variations. What do you think?

Reply:
Yes, we could probably just keep the first link. I summed up the different links for a region, because it didn't look like they represent the same products. If all of them are contained in the first figure, summing them up again would obviously be redundant.

Reply:
Let's try it as it is now.